### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by MakeMake23**

### CODE REVIEW

File: docs/02_expressions.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ The rest of this section explains the basic expressions in HULK.
 
 ## Arithmetic expressions
 
-HULK defines three types of literal values: **numbers**, **strings**, and **booleans**.
+HULK defines three types of literal values: **numbers**, **strings** and **booleans**.
```
Comment (jpconsuegra) [inline docs/02_expressions.md]:
```suggestion
HULK defines three types of literal values: **numbers**, **strings**, and **booleans**.
```

In English, the comma is mandatory there.

File: docs/10_protocols.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ type Person {
 let x : Hashable = new Person() in print(x.hash());
 ```
 
-Anywhere you can annotate a symbol with a type (variables, attributes, function,method and type arguments, and return values), you can also use a protocol. For the purpose of type inference, protocols are treated as types.
+Anywhere you can annotate a symbol with a type (variables, attributes, function, method, type arguments and return values), you can also use a protocol. For the purpose of type inference, protocols are treated as types.
```
Comment (jpconsuegra) [inline docs/10_protocols.md]: I think the original version was correct except for the missing space.

```suggestion
Anywhere you can annotate a symbol with a type (variables, attributes, function, method and type arguments, and return values), you can also use a protocol. For the purpose of type inference, protocols are treated as types.
```

**Review by jpconsuegra — CHANGES_REQUESTED** [review]
Excluding the two changes I commented on, I think the rest can be committed.

File: docs/10_protocols.md (hunk 2 above)
Comment (MakeMake23) [reply docs/10_protocols.md]: You are right, I didn't realize "arguments" refers to both "method arguments" and "type arguments".

**Pull request merged**